itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/WarehouseInstance.java

```diff
@@ -380,6 +381,37 @@ public Table getTable(String dbName, String tableName) throws Exception {
     }
   }
 
+  /**
+   * Get statistics for given set of columns of a given table in the given database
+   * @param dbName - the database where the table resides
+   * @param tableName - tablename whose statistics are to be retrieved
+   * @param colNames - columns whose statistics is to be retrieved.
```

Comment:
colNames is not parameter.

Reply:
Removed. It used to be a parameter in an earlier version. Forgot to remove it. Thanks for catching.